subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -171,10 +171,25 @@
                                     <li><a href="/userguide/feature_lifecycle.html">Feature Lifecycle</a></li></ul></li>
                             <li><a class="nav-dropdown" data-toggle="collapse" href="#managing-dependencies" aria-expanded="false" aria-controls="managing-dependencies">Managing Dependencies</a>
                                 <ul id="managing-dependencies">
-                                    <li><a href="/userguide/artifact_dependencies_tutorial.html">Dependency Management Basics</a></li>
-                                    <li><a href="/userguide/dependency_management.html">Advanced Dependency Management</a></li>
-                                    <li><a href="/userguide/dependency_types.html">Dependency Types</a></li>
-                                    <li><a href="/userguide/repository_types.html">Repository Types</a></li></ul></li>
+                                    <li><a class="nav-dropdown" data-toggle="collapse" href="#dependency-management-tutorials" aria-expanded="false" aria-controls="dependency-management-tutorials">Tutorials</a>
+                                        <ul id="dependency-management-tutorials">
+                                            <li><a href="/userguide/artifact_dependencies_tutorial.html">Dependency Management for Java Projects</a></li>
```

Comment:
This might belong better under "JVM Projects" bit of navigation. WDYT?

Reply:
Yes and no. At the moment the page is a mixture between "explaining what dependency management is" and giving some Java-based examples. I think it could fit underneath either section. How about we rewrite the "Introduction to Dependency Management" first and then we what we can remove from the page? But I don't feel too strongly about where it should live right now. At least under "JVM Projects" we already have a Tutorials section.